Vagrantfile

```diff
@@ -30,6 +30,7 @@ SCRIPT
     ansible.compatibility_mode = "2.0"
     ansible.install_mode = "pip_args_only"
     ansible.extra_vars = { ansible_python_interpreter:"/usr/bin/python3" }
+    ansible.pip_install_cmd = "curl https://bootstrap.pypa.io/pip/2.7/get-pip.py | sudo python" 
```

Comment:
Since we're using `python3` in this project (see line above and the declaration for `pip3` in `group_vars/all`), why go with `2.7` here?

Reply:
When I first tried changing to 3.0, the URL returned a 404 error. It seems like after updating the pip version in the group_vars it is working properly now, however. (I don't know why that would change the URL response, but it is working now!) I've updated in a fix-up - thank you for the catch!

Reply:
Looking at the Python version available inside the VM, it turns out to be 3.8.10:

```console
❯ vagrant ssh -c 'python3 --version'
Python 3.8.10
```

So using a 3.0 version of pip here is incorrect. Let's do this instead, to parse the latest version of pip via `python3` instead of `python`, and installing `python3-distutils` which are needed for the pip installation via `python3`:

```diff
diff --git a/Vagrantfile b/Vagrantfile
index b1a3aed..a82df10 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -24,13 +24,15 @@ Vagrant.configure(2) do |config|
       echo "cd /vagrant" >> "/home/vagrant/.bashrc"
     fi
 
+    sudo apt-get install -qq -y python3-distutils
+
 SCRIPT
 
   config.vm.provision "ansible_local" do |ansible|
     ansible.compatibility_mode = "2.0"
     ansible.install_mode = "pip_args_only"
     ansible.extra_vars = { ansible_python_interpreter:"/usr/bin/python3" }
-    ansible.pip_install_cmd = "curl https://bootstrap.pypa.io/pip/3.0/get-pip.py | sudo python" 
+    ansible.pip_install_cmd = "curl https://bootstrap.pypa.io/get-pip.py | sudo python3"
     ansible.pip_args = "ansible==#{ANSIBLE_VERSION}"
     ansible.playbook = "deployment/ansible/fb-gender-survey.yml"
     ansible.galaxy_role_file = "deployment/ansible/roles.yml"
```

This was recently done for Travis's Breakable Toy project:

https://github.com/azavea/bike-route-breakable-toy/blob/develop/Vagrantfile

Reply:
That's excellent!